src/main/java/org/rumbledb/expressions/control/TypeswitchCase.java

```diff
@@ -0,0 +1,50 @@
+package org.rumbledb.expressions.control;
+
+
+import org.rumbledb.expressions.Expression;
+import org.rumbledb.expressions.flowr.FlworVarSequenceType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This is a helper class that organizes the children expressions of a typeswitch expression.
+ * From a tree perspective, all expressions in there are considered
+ * to be direct children of the TypeswitchExpression.
+ */
+public class TypeswitchCase {
```

Comment:
Is this name intentional or should the s in TypeSwitchCase be capitalized?

Reply:
I am trying to unify with the original spec (https://www.w3.org/TR/xquery-30/#id-typeswitch) where this is capitalized as Typeswitch

Reply:
Alright that sounds great! Thanks for the clarification